### PR TITLE
dep: Bump grpc_health_probe to v0.4.24

### PR DESCRIPTION
**Pull request by ferozsalam**
Contains updated deps that clear up some CVEs.

### CODE REVIEW

**Review by sayboras — APPROVED** [review]
Looks legit, I came across similar update in envoy repo.

**Review by rolinh — APPROVED** [review]

**Review by chancez — APPROVED** [review]

File: images/hubble-relay/download-grpc-health-probe.sh [hunk 1]
```diff
@@ -9,13 +9,13 @@ set -o pipefail
...
 
 # renovate: datasource=github-release-attachments depName=grpc-ecosystem/grpc-health-probe
-grpc_health_probe_version="v0.4.21"
+grpc_health_probe_version="v0.4.24"
 
 declare -A grpc_health_probe_sha256
 # renovate: datasource=github-release-attachments depName=grpc-ecosystem/grpc-health-probe digestVersion=v0.4.21
-grpc_health_probe_sha256[amd64]="dbf3d1ea952ffe6dac5405b6d4cc0e630476272ef17ece02e020c55430f471ce"
+grpc_health_probe_sha256[amd64]="7e564681110ee4563637457b91e42f62f96b79618a835bb05ae2305acdcc3db0"
 # renovate: datasource=github-release-attachments depName=grpc-ecosystem/grpc-health-probe digestVersion=v0.4.21
```
Comment (chancez) [inline images/hubble-relay/download-grpc-health-probe.sh]: Should we be updating digestVersion?

**Review by chancez — CHANGES_REQUESTED** [review]
Should the renovate comment be updated as well? Also, why has renovate tried to update this if there's been multiple releases since it's been added?

**Comment by ferozsalam** [comment]
> Should the renovate comment be updated as well? Also, why has renovate tried to update this if there's been multiple releases since it's been added?

I've updated the Renovate comment and pinged @glrf yesterday to review the Renovate behaviour so that we don't have to do this manually in the future – I am also not sure why Renovate isn't picking the updates up automatically.

**Comment by glrf** [comment]
I think the reason why renovate wasn't picking it up was because I forgot to add a file match here: https://github.com/cilium/cilium/blob/main/.github/renovate.json5#L499-L501 (sorry about that :sweat_smile: )

**Review by aanm — APPROVED** [review]

**Review by chancez — APPROVED** [review]

**Pull request merged**